HW_03/hw_3_01.py

```diff
@@ -0,0 +1,20 @@
+# Задайте список из нескольких чисел. Напишите программу, которая найдёт сумму элементов списка, стоящих на нечётной позиции.
+#  Пример:
+# [2, 3, 5, 9, 3] -> на нечётных позициях элементы 3 и 9, ответ: 12
+
+from random import randint
+
+
+n = int(input('Размер генерируемого списка: '))
+
+lst = []
+
+for i in range(1, n+1):
+    lst.append(randint(-n*n, n*n))
+print('Сгенерирован список:\n',lst)
+
+sum = 0
+for i in range(len(lst)):
```

Comment:
```suggestion
for i in range(n):
```
Зачем делать вычисления, если вас уже есть такая переменная?